README.md

```diff
@@ -1 +1,31 @@
-# som_timer_be
\ No newline at end of file
+# som_timer_be
+
+## Endpoints
+
+#### GET  '/api/timers/{timer_id}
```

Comment:
Thanks so much for creating this documentation! Am I correct that with our MVP version of this app that does not include users, we will only have one timer total, so we would always use the id of 1 for the endpoint? That seems fine with MVP, but I'm wondering how this would be adapted for when there are users, because it makes more sense to me from a FE perspective to get the timer based on userId, since it would be the timer associated with whatever user is logged in (and we would have gotten that user ID info on login). Were you thinking something similar for how that would work if we eventually went in that direction?

Reply:
So our initial thought is that Timer with id of 1 will be the default timer that every user has as a template for their use. When we get into users down the road, we can alter how we do timers but for now it seems easiest to just access the single timers we will need for MVP.

Reply:
This makes sense to me for MVP. I also really appreciate the conversation going n here